python/src/usecases/fireSatellite_function.py

```diff
@@ -0,0 +1,534 @@
+			   
+    c : Speed of light, constant
+        c = 2.9979e8 m/s
+	
+    omega_max : Maximum rotational velocity of reaction wheel, constant
+        omega_max = 6000 rpm
+        
+    n : Number of reaction wheels that could be active, constant
+        n = 3
+        
+    delta_theta_slew : Slewing time period, constant
+        delta_theta_slew = 760 s
+        
+    As : Area reflecting radiation, constant
+        As =  13.85 m^2
+        
+    i : Sun incidence angle, constant
+        i = 0 deg
+        
+    M :  Magnetic moment of earth, constant
+        M =  7.96e15 A.m^2
+        
+    rho : Atmospheric density, constant
+        rho =  5.1480e-11 kg/m^3
+        
+    A : Cross-sectional in flight direction, constant
+        A =  13.85 m^2
+        
+    Phold : Holding power, constant
+        Phold = 20 W
+        
+    mu :  Earth gravity constant
+        mu =  398600.4418e9 m^3/s^2
+        
+    Id : Inherent degradation of array, constant
+        Id = 0.77
+        
+    t : Thickness of solar panels, constant
+        t =  0.005 m
+        
+    n_sa : Number of solar arrays, constant
+        n_sa = 3
+        
+    epsilon_deg : Degradation in power production capability, constant
+        epsilon_deg = 0.0375 percent per year
+        
+    LT : Lifetime of spacecraft, constant
+	    LT = 15 years
+        
+    r_lw : Length to width ratio of solar array, constant
+        r_lw =  3
+
+    D : Distance between panels, constant
+        D = 2 m
+    
+    I_bodyX : Inertia of body, X axis, constant
+        I_bodyX = 6200 kg.m^2
+
+    I_bodyY : Inertia of body, Y axis, constant
+        I_bodyY =  6200 kg.m^2
+	
+    I_bodyZ : Inertia of body, Z axis, constant
+        I_bodyZ =  4700 kg.m^2
+ 
+    rho_sa : Average mass density to arrays, constant
+        rho_sa =  700 kg.m^3
+        
+    eta :  Power efficiency, constant
+        eta =  0.22
+
+    phi_target : Target diameter, constant
+        phi_target = 235000 m
+		
+    RE : Earth radius, constant
+        RE = 6378140 m  
+    
+    tolFPI : Tolerance on Fixed Point Iteration used in the multidisciplinary analysis
+        tolFPI = 1e-3  (deterministic)
+
+    maxFPIIter : Maximum number of iterations of Fixed Point Iteration used in the multidisciplinary analysis
+        maxFPIIter = 50  (deterministic)
+    
+	
+	
+    Examples
+    --------
+    >>> from openturns.usecases import fireSatellitefunction
+    >>> # Load the FireSatellite model model
+    >>> m = fireSatellitefunction.FireSatelliteModel()
+    """
+
+    def __init__(self):
+        
+        # dimension
+        self.dim = 9
+        
+        # Altitude 
+        self.H = ot.Normal(18e6,1e6)
+        
+        # Power other than ACS
+        self.Pother = ot.Normal(1000.,50.)
```

Comment:
I suggest to use Black on the script (https://pypi.org/project/black/) to prettyfy it. For example, the script `ot.Normal(1000.,50.)` will become `ot.Normal(1000.0, 50.0)`. This is easy and will be make the script match PEP8.

Reply:
Thanks. Black has been run on all the scripts.